gms/gossiper.cc

```diff
@@ -1763,38 +1763,84 @@ future<> gossiper::start_gossiping(int generation_nbr, std::map<application_stat
     });
 }
```

Comment:
changelog: s/talks to/attempts to talk to/

Reply:
OK